src/deepsparse/pipeline.py

```diff
@@ -119,6 +120,7 @@ def __init__(
         scheduler: Scheduler = None,
         input_shapes: List[List[int]] = None,
         alias: Optional[str] = None,
+        context: Optional["Context"] = None,
```

Comment:
why are the quotes needed here? looks like Context is already imported

Reply:
Good catch, thanks!